kuma/javascript/src/error-boundaries.jsx

```diff
@@ -1,24 +1,31 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 
+import GAProvider from './ga-provider.jsx';
 import { gettext } from './l10n.js';
 
 export class AppErrorBoundary extends React.Component {
     state = { error: null };
 
+    static contextType = GAProvider.context;
+
     static getDerivedStateFromError(error) {
         return { error };
     }
 
-    logError() {
-        // https://github.com/mozilla/kuma/issues/5833
-        // When we start to use Sentry, change the method signature to
-        // logError(error, errorInfo) {
+    logError(boundaryName) {
+        // https://developers.google.com/analytics/devguides/collection/analyticsjs/events#event_fields
+        this.context('send', {
+            hitType: 'event',
+            eventCategory: 'errorboundary',
+            eventAction: boundaryName,
+            eventLabel: document.location.href
```

Comment:
Will `document` always be defined? I don't know how this React component gets access to the `document` object. Must be some React magic under the hood?

Reply:
An error boundary can only ever happen in a browser. There, the `document` and `window` globals are always available. At least I hope so :)

Reply:
If my comment on line 10 above makes sense/is valid, then we can do the following here:

```
ga('send', {
  hitType: 'event',
  eventCategory: 'errorboundary',
  eventAction: boundaryName,
  eventLabel: document.location.href
});
```